README.md

```diff
@@ -36,15 +37,13 @@ Once you have the project open, you can run it within an emulator or on any Andr
 Because much of the application logic takes place in WebViews, you'll likely want to set up [Remote debugging on Android with Chrome](https://developer.chrome.com/devtools/docs/remote-debugging).
 
 #### Web
-Each fragment within `webmaker-android` is actually just a web page! You can find all of the js, css, and static assets in the `./www_src/` directory. Static files in `./www_src/static/` will be copied to the main directory during build.
+Each fragment within `webmaker-android` is actually just a web page! You can find all of the js, css, and static assets in the `webmaker-core` module. Static files in `./node_modules/webmaker-core/src/dest/` will be copied up to this Android wrapper as part of `npm run build`.
 
-To run and develop in a web browser without testing on device, simply run
+**NOTE:**
 
-```bash
-npm start
-```
+For local development, it's recommended to use `npm link` with a local copy of `webmaker-core`, in which you'll do any webview related work separately.
 
-Please note that this is only really useful for basic content and style changes; a lot of functionality is likely to fail due to the absence of Android APIs and other factors.
+When changes are compiled in `webmaker-core` you'll need to run `npm run copy:core` before building in Android Studio. Alternatively, you can create a symbolic link from `app/src/main/assets/www/` to `./node_modules/webmaker-core/dest/` to avoid having to run this extra command.
```

Comment:
also maybe instructions on how to symlink, or at least what a symlink is https://en.wikipedia.org/wiki/Symbolic_link